coding-concepts.js

```diff
@@ -67,7 +68,7 @@ class Learn {
   }
 }
 var learnStudent = new Learn("George")
-// console.log(learnStudent)
+ console.log(learnStudent)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer:Learn student george
+// b) Verify and explain: console logged the entire class and with every assigned key value. Honestly still a little unsure on how this works. 
```

Comment:
This will become more familiar with more practice.  This was a stretch problem, so great job in trying!